src/blurbehind.cc

```diff
@@ -1,43 +1,46 @@
+#define WIN32_LEAN_AND_MEAN
 #include <dwmapi.h>
-#include <node.h>
+#include <napi.h>
+#include <uv.h>
 #include <node_buffer.h>
 
-void blurbehind(const v8::FunctionCallbackInfo<v8::Value> &args)
+Napi::Value blurbehind(const Napi::CallbackInfo& info)
 {
-	const HWND handle = *reinterpret_cast<HWND *>(node::Buffer::Data(args[0].As<v8::Object>()));
+	const HWND handle = (HWND)info[0].ToNumber().Int64Value();
```

Comment:
I am not familiar with NAPI but I believe we want info[0].As<Napi::Buffer>() and we can then access the underlying data.